canonical-web-and-design/maas-ui
# refactor(ui): wrap routes in component prop

## Done

- Update the routes to wrap the content in a `component` prop.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Load the UI and have a poke around the React pages. They should load as expected.

## Fixes

Fixes: #3507.
